content/en/dora_metrics/setup/deployments.md

```diff
@@ -135,7 +147,7 @@ If the [GitHub integration][1] is not already installed, install it on the [GitH
 
 When configuring the GitHub application:
 1. Select at least **Read** repository permissions for **Contents**.
-2. Subscribe at least to **Push** events.
+2. Subscribe at least to **Push**, **PullRequest** and **PullRequestReview** events.
```

Comment:
IIRC here they need both a new permission and to subscribe to a new event. I think we need to modify also point 1

Reply:
Right, updated!